v0/destinations/statsig/transform.js

```diff
@@ -0,0 +1,24 @@
+const {
+  defaultPostRequestConfig,
+  defaultRequestConfig
+} = require("../../util");
+const { ENDPOINT } = require("./config");
+
+function process(event) {
+  const { message, destination } = event;
```

Comment:
As I see only a few message types are supported, can you validate and throw an "unsupported message type" error?

Reply:
Ok, I have addressed this in https://github.com/rudderlabs/rudder-transformer/pull/933/commits/003699807f61fc9a915947e9877dfa2ae080a578

Reply:
@daniel-statsig seems like you've missed importing the `CustomError` class

Reply:
Ah, you are correct. I have fixed this add added a test case for this flow. Commit is https://github.com/rudderlabs/rudder-transformer/pull/933/commits/1428a6e2aa7e043c93e2b66a8f9e3ab953902704. Thank you.